### ISSUE TITLE
Doesn't seem to work with newer BT devices

### ISSUE THREAD

**Issue by JeroenBots** [HelloVolla/ubuntu-touch-beta-tests#69]
When try to connect with LE-Bose Micro Soundlink it will connect, but disconnect within a few seconds, before you hear any sound.

**Comment by JeroenBots** [comment]
**Details about installed OS version**
Ubuntu-Touch 16.04 on Volla Phone

Info about which OTA/Image you are reporting this issue for.
OTA-14 Stable

Describe the bug
When try to connect with LE-Bose Micro Soundlink it will connect, but disconnect within a few seconds, before you hear any sound. 
If I try to connect older BT speakers right after they pair without any problem


![connecting](https://user-images.githubusercontent.com/21966556/99305389-00294280-2854-11eb-928a-7b12739c15ee.png)
![connected](https://user-images.githubusercontent.com/21966556/99305443-13d4a900-2854-11eb-99af-8251caf06d6e.png)
![LostConnectionAfterAfewSec](https://user-images.githubusercontent.com/21966556/99305439-133c1280-2854-11eb-9002-d18669739130.png)
![SignalIsExcellent](https://user-images.githubusercontent.com/21966556/99305432-120ae580-2854-11eb-91ad-4275397c85ca.png)

**Comment by Mattia990** [comment]
Me too with a pair of true wireless bluetooth called Pamu Scrolls, Bluetooth 5.0

**Comment by Bsoupe** [comment]
My JBL FLIP4 works fine with the Volla Phone and OTA-14 stable. My Peugeot Headunit can establish a connection but it doesn't share contacts and doesn't allow calls over the phone. My Sailfish device works good with this device (except screen mirroring)

**Comment by mateosalta** [comment]
I found a fix for bose devices https://askubuntu.com/questions/833322/pair-bose-quietcomfort-35-with-ubuntu-over-bluetooth

so in sort (enable root r/w first)

sudo nano /etc/bluetooth/main.conf

add

ControllerMode = bredr
AutoEnable=true
FastConnectable = true


Then select "forget this device"  on the device, and make sure to reboot before testing again
(also try dev for cars it seems to have fixes in for that)

**Comment by kwoot** [comment]
Input from Florian regarding the reply from Mateo: 

The problem is that that fix will disable Bluetooth Low Energy mode, and unless you do those manual steps every time you want to pair an affected device, you can only vote for leaving LE off. That will affect battery life for many smaller devices, and devices that only speak LE cannot be paired at all. To me that sounds not like a proper fix

(And since our rootfs is readonly by default doing those steps manually sucks balls, if I can say so)

**Comment by mateosalta** [comment]
yeah, i wonder if we can find some kind of 'pair in' workaround, as it seens only pairing is affected for bose, and once paired, le will work if that is reverted

**Comment by wurzer** [comment]
Still valid ticket?

**Comment by Flohack74** [comment]
@wurzer yes still very valid but: We are about to increase Bluez version to 5.53 (equal to Ubuntu focal)...

For those of you who want to test pairing issues already before, here are the instructions:

Ok so here are the steps:
1. You must be on the devel channel
2. You must be fully upgraded to latest devel - check system-settings upgrades for new stuff and install it
3. You will need 2 consoles open to the device (adb or ssh, whatever is preferred/works)
Console 1: `sudo service bluetooth stop`
Console 2: `sudo ubports-qa install xenial_-_even-newer-bluez`
Console 1: When installation stops with the bluetooth.conf file, again `sudo service bluetooth stop`
Console 2: `sudo reboot` when finished

**Comment by JamiKettunen** [comment]
I believe that newer `bluez` version change was reverted on 16.04; could someone check on Ubuntu Touch 20.04 if this issue is still present?

**Comment by wurzer** [comment]
@JamiKettunen dev chennel?
